week6/Users.cs

```diff
@@ -0,0 +1,57 @@
+using Dapper;
+using MySql.Data.MySqlClient;
+namespace HackYourFuture.Week6;
+
+
+public record User(int Id, string Fullname, string Address);
+
+public interface IUserRepository
+{
+    Task<IEnumerable<User>> GetUsers();
+    Task<User> PostUser(User user);
+    Task<int> UpdateUser(int id, User user);
+    Task<int> DeleteUser(int id);
+}
+
+
+public class UserRepository : IUserRepository
```

Comment:
Similar to the `ProductRepository`, this could be two separate files, `UserRepository.cs` and `IUserRepository.cs`

Reply:
Thank you for great feedbacks.